admin_guide/high_availability.adoc

```diff
@@ -580,6 +588,51 @@ the list of VIPs.  The vrrp-ids may have values in the range 1..255.
 
 When there are multiple ipfailover deployment configuration care must be taken to specify `--vrrp-id-offset` so that there is room to increase the number of VIPS in the deployment configuration and none of the vrrp-id ranges overlap.
 
+[[admin-guide-high-availability-configuring-more-than-254]]
+=== Configuring IP failover for more than 254 addresses
+
+By default, IP failover has a limitation of managing only 254 VIP addresses. You
+can group sets of VIPs for VRRP instances using the `virtual-ip-groups` option
+to define the number of VIP groups available for each VRRP instance when
+cxref:../admin_guide/high_availability.adoc#configuring-ip-failover[configuring
```

Comment:
s/cxref/xref 

The path should be xref:configuring-ip-failover 
(It points to another section in this same file.)